packages/components/src/Combobox/Combobox.types.ts

```diff
@@ -15,7 +15,7 @@ export interface ComboboxProps {
   /**
    * The Chip heading for the trigger
    */
-  readonly heading: string;
+  readonly heading?: string;
```

Comment:
Now that i'm looking at this closely. should this be called heading? I get that it is called heading on the chip but in the event (future proofing) that we changed our mind and say it's not a chip anymore, does heading still make sense as a combobox prop? Basically decoupling what is exposed to the consumer and what it is doing behind the scenes.

If we look at all the _input_-driven things we have. Inputs have placeholders, Checkbox/Radios have labels, should this just be called label? (Again, decoupling it from what is being implemented behind the scenes and what would the consumer reach for if they do need to add it)

Reply:
Yup good point. I've been going back and forth about using `label` or `placeholder` since future iterations will likely include Selects and Inputs, etc.  
Let's go with `label` since a placeholder could be an example of an expected value, and that's not what this prop is doing.

Reply:
Sounds good! This looks good to me so just need that 1 change and we solid